scanpipe/tests/test_models.py

```diff
@@ -2352,6 +2356,22 @@ def test_scanpipe_codebase_resource_queryset_elfs(self):
         self.assertTrue("e" in paths)
         self.assertTrue("a" in paths)
 
+    def test_scorecard_models(self):
```

Comment:
PLease also add a pipeline integration test running the full pipeline. See the examples under: [`PipelinesIntegrationTest` class](https://github.com/nexB/scancode.io/blob/main/scanpipe/tests/test_pipelines.py#L502)

Reply:
done https://github.com/nexB/scancode.io/pull/1294/commits/df5041682d6aa69156c4ec08be20ba96aa8cec5f